pkg/spec/populators.go

```diff
@@ -23,9 +23,26 @@ import (
 
 	log "github.com/Sirupsen/logrus"
 	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/client-go/pkg/api"
 	api_v1 "k8s.io/client-go/pkg/api/v1"
+	"strconv"
```

Comment:
can you arrange the imports similar to the comment here https://github.com/kedgeproject/kedge/pull/149#issuecomment-315051859

Reply:
fixed